### PR TITLE
Update building-a-react-native-app.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/yaml-quick-start/building-a-react-native-app.md [hunk 1]
```diff
@@ -223,8 +221,10 @@ workflows:
           success: true               # To receive a notification when a build succeeds
           failure: false              # To not receive a notification when a build fails
       app_store_connect:
-        apple_id: $APPLE_ID
-        password: $APP_SPECIFIC_PASSWORD
+          api_key: Encrypted(...)           # Contents of the API key, can also reference environment variable such as $APP_STORE_CONNECT_PRIVATE_KEY
+          key_id: 3MD9688D9K                # Alphanumeric value that identifies the API key, can also reference environment variable such as $APP_STORE_CONNECT_KEY_IDENTIFIER
+          issuer_id: 21d78e2f-b8ad-...      # Alphanumeric value that identifies who created the API key, can also reference environment variable such as $APP_STORE_CONNECT_ISSUER_ID
+          submit_to_testflight: true        # Optional boolean, defaults to false. Whether or not to submit the uploaded build to TestFlight to automatically enroll your build to beta testers.  
 ```
```
Comment (helinanever) [inline content/yaml-quick-start/building-a-react-native-app.md:228]: some of these values you also have under environment variables

Comment (icarusdust) [reply content/yaml-quick-start/building-a-react-native-app.md:228]: updated PR

Comment (icarusdust) [reply content/yaml-quick-start/building-a-react-native-app.md:228]: or should we also remove the comments ?

Comment (helinanever) [reply content/yaml-quick-start/building-a-react-native-app.md:228]: I'm fine keeping the comments

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**